assets/src/settings-page/analytics.js

```diff
@@ -3,14 +3,15 @@
  */
 import PropTypes from 'prop-types';
 import { v4 as uuidv4 } from 'uuid';
+import { ANALYTICS_VENDORS_LIST as AnalyticsVendors } from 'amp-settings';
```

Comment:
I think we should follow the same convention as in other JS files, i.e. use the uppercase external constant names. Also, as I believe @westonruter noted earlier, using title case for a variable name (`AnalyticsVendors`) may suggest it is a React component while it's not.

Reply:
@delawski That means you are suggesting it should be `ANALYTICSVENDORS` right?

Reply:
@delawski It stands resolved. 
Will take care of this in the future.